packages/rlc-common/src/interfaces.ts

```diff
@@ -106,7 +106,7 @@ export interface ApiVersionInfo {
   isCrossedVersion?: boolean;
 }
 
-export type ApiVersionPosition = "path" | "query" | "both" | "none";
+export type ApiVersionPosition = "path" | "query" | "url" | "duplicate" | "none";
```

Comment:
not sure we could have a better name to distingush the `path` and `url`?

Reply:
only the default value for api version defined in url and query can actually has effect right now, for the path parameter, currently we don't have a way to set it, at least in RLC. 

For the name, I don't have a strong opinion, maybe @joheredi could help to decide ? Thanks

Reply:
@joheredi do you have any comments on this ? feel free to let me know. Thanks

Reply:
I wouldn't want to take a breaking change to rename path, also I think the path is a standard term. For url I tink I'd be fine to name it baseUrl, only if we don't need to do a breaking change for that.

Reply:
I am not sure about the breaking here, I think ApiVersionPosition should be used internally? but I am fine to go with path and baseurl.